src/current/v23.1/alter-table.md

```diff
@@ -911,6 +911,50 @@ $ cockroach demo bank
 (1 row)
 ~~~
 
+##### Move a column from one column family to another
+
+To move a column from one column family to another column family, create a temporary, [non-visible](create-table.html#not-visible-property) [stored computed column](computed-columns.html) in the target column family, then rename the columns. Once this succeeds, you can drop the original, now renamed column.
```

Comment:
I think it would be good to add a line about why one might want to do this.  I didn't know why one would want to do this but DOC-7566 has it: "say I create a table with just the default primary family. Then later I realize 3 of the 10 columns get frequently updated so I want to move them to their own family."

In fact I would really prefer even more detail than that: _why_ do I care that only 3/10 of the columns get frequently updated?  Is this a substantial optimization? if so, how substantial (roughly)?

Reply:
ok maybe the 2nd question about perf would be answered by the link to https://www.cockroachlabs.com/docs/stable/column-families.html#default-behavior

but I still think it would be good to add a line about the "later I realize 3/10 are frequently updated" etc